code/http4s/src/main/scala/com/avast/datadog4s/extension/http4s/DatadogMetricsOps.scala

```diff
@@ -32,14 +36,15 @@ object DatadogMetricsOps {
       headersTime
         .record(
           Duration.ofNanos(elapsed),
-          Tag.of("method", method.name) :: classifier.toList.flatMap(classifierTags): _*
+          methodTag.tag(method.name) :: classifier.toList.flatMap(classifierTags): _*
```

Comment:
wouldn't it be better if `methodTagger` would be of a type `Tagger[org.http4s.Method]`? (similar to other types)

Reply:
for method it does make sense